setup.sh

```diff
@@ -36,11 +36,19 @@ if [[ "$unamestr" == 'Darwin' ]]; then
   fi  
 fi
 
+echo '[CONFIG] Configure Jadx'
+if [[ "$unamestr" == 'Darwin' ]]; then
+    sed -i '' "s#DEFAULT_JVM_OPTS=.*#DEFAULT_JVM_OPTS='\"-Xms128M\" \"-Xmx4g\" \"-XX:+UseG1GC\" \"-Dlogback.configurationFile=${PWD}/jadx.xml\"'#" ./StaticAnalyzer/tools/jadx/bin/jadx
+else
+    sed -i "s#DEFAULT_JVM_OPTS=.*#DEFAULT_JVM_OPTS='\"-Xms128M\" \"-Xmx4g\" \"-XX:+UseG1GC\" \"-Dlogback.configurationFile=${PWD}/jadx.xml\"'#" ./StaticAnalyzer/tools/jadx/bin/jadx
+fi
+
```

Comment:
What about Windows?
This is a core feature and major change. Should work across all OS properly.

Reply:
need to find a way to reduce logs on windows

Reply:
log reduces on windows too and jadx.xml moved to jadx bin dir